sql/executor.go

```diff
@@ -54,10 +54,13 @@ func (e *Executor) SetNodeID(nodeID proto.NodeID) {
 // Execute the statement(s) in the given request and return a response.
 // On error, the returned integer is an HTTP error code.
 func (e Executor) Execute(args driver.Request) (driver.Response, int, error) {
+	txnTimestamp := time.Now()
```

Comment:
It's both: the statement timestamp and transaction timestamp are the same by default until we read a new value for txnTimestamp from the session Transaction. But I agree it would be clearer to have two separate variables.